cli_reference/configuring-cli.adoc

```diff
@@ -0,0 +1,9 @@
+[id='cli-configuring-cli']
+= Configuring the CLI
+include::modules/common-attributes.adoc[]
+:context: cli-configuring-cli
+
+toc::[]
+
```

Comment:
Where do you explain how to install the CLI? I made a `install-cli.adoc` module for the install guide with basic instructions, but only the admin who installs a cluster will encounter that. I might make this topic "Installing and configuring the CLI" and include the installation steps and a guide to logging into the command line before you configure it. (Or make another assembly about installing and logging in.)

If you already have a module about installing the cli, will you let me know so we can combine them?

Reply:
@kalexand-rh I've been adding my sections a bit out of order, so I haven't created that module yet. But there will eventually be a getting started section that will include installing and logging in to the CLI. I'll ping you once I do add this.